sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -63,6 +102,24 @@ export interface OnMessage {
     (message: ServiceBusMessage): Promise<void>;
 }
 
+export { ProxySettings }
+
+// @public
+export interface Queue extends QueueOptions {
+    _?: any;
+    AccessedAt?: string;
```

Comment:
I think we typically start object properties with a lowercase letter. It looks like this is the case for all the existing interfaces (and some new).

Since we have control over what the user passes in prior to serialization, we could transform the user input into what the service expects the properties to be. Likewise for when we get a response from the service.

Reply:
Per offline discussions with @bterlson, we want to use camelCasing for the json object that comes in as user input as that is what javascript users would expect